src/ui/components/app/exporters/exportersCommon.ts

```diff
@@ -11,7 +11,7 @@ export function setupExporterConfigForm<T extends OutputVendorName>(exporterName
 
   const validated = ref(true);
   const changed = ref(false);
-  const readyToSave = computed(() => validated && changed);
+  const readyToSave = computed(() => validated.value && changed.value);
```

Comment:
This was the bug. Since `validated` and `changed` themself are `Object`s, the `readyToSave` was always `true`.